src/common/__tests__/BaseAnnotator-test.ts

```diff
@@ -156,6 +156,16 @@ describe('BaseAnnotator', () => {
             expect(annotator.containerEl.classList).not.toContain(CSS_CONTAINER_CLASS);
         });
 
+        test('should remove the loaded class from annotation layer', () => {
```

Comment:
Should we test the other annotation mode classes get removed as well?

Reply:
Yep will do